makejdk-any-platform.sh

```diff
@@ -54,8 +54,8 @@ writeConfigToFile
 echo "$@" > ./workspace/config/makejdk-any-platform.args
 
 # Let's build and test the (Adoptium) OpenJDK binary in Docker or natively
-if [ "${BUILD_CONFIG[USE_DOCKER]}" == "true" ] ; then
-  buildOpenJDKViaDocker
+if [ ! "${BUILD_CONFIG[USE_DOCKER]}" == "false" ] ; then
```

Comment:
This line is confusing me - why has it been changed from true to false?
This looks like it will NOT invoke `buildOpenJDKInNativeEnvironment` when `USE_DOCKER` is false.

Reply:
It seems that thsi comemtn was not send yeserday. So long story short.

Originally BUILD_CONFIG[USE_DOCKER] was true/false . Now it is  false/podman/docker.
buildOpenJDKViaDocker was origianlly invoked if it was true, not it is invoked if it is not false  - so if it is podman/docker.    Does it gave more sense

Reply:
Yep thanks. I think I missed the `!` when I did my review :-)

Reply:
ty!